server/controllers/annotation_controller.js

```diff
@@ -54,10 +54,9 @@ export const getReplies = (user, parentId) => {
 };
 
 // PRECONDITION: user is not null.
```

Comment:
We can just remove this comment, since there's no user being passed in - or just say user checking is happening elsewhere.